syntax/supercollider.vim

```diff
@@ -39,6 +39,7 @@ syn match	scSymbol "\w\+:"
 
 syn region  scString start=+"+ skip=+\\\\\|\\"+ end=+"+
 
+syn match	scChar	"\$[^ ]"
```

Comment:
This definitely makes sending character literals such as `NetAddr.localAddr.sendMsg("/hello", $T)` work.